pages/docs/configuration/configuration-mapping.md

```diff
@@ -50,61 +56,104 @@ Furthermore, mappings come int two types of `constraint`: `consistent` and `cons
+  * `rbf-global-iterative`, which assembles the system matrix initially and solves the resulting system iteratively in each coupling iteration. This method can use differenten [execution backends](configuration-mapping.html#execution-backends). However, the CPU version relies on the linear-algebra library PETSc, which is an optional dependency of preCICE. The PETSc implementation is fully mpi-parallel and uses sparse linear algebra to solve the resulting system. As a consequence, care has to be taken when configuring the basis-function for the mapping: it should only be used with compactly supported basis-functions and the configured support-radius or shape-parameter should only cover a limited number of vertices in radial direction (e.g. 10 vertices) such that the resulting linear system is sparse. Details about this implementation can be found in [Florian's thesis (pages 37 ff.)](https://elib.uni-stuttgart.de/bitstream/11682/10598/3/Lindner%20-%20Data%20Transfer%20in%20Partitioned%20Multi-Physics%20Simulations.pdf).
 
-The configuration might look like this:
+{% note %}
+For global rbf methods, the interpolation problem (or rather the polynomial QR system) might not be well-defined if you map along an axis-symmetric surface. This means, preCICE tries to compute, for example, a 3D interpolant out of 2D information. If so, preCICE throws an error. In this case, you can restrict the interpolation problem by ignoring certain coordinates, e.g. `x-dead="true"` to ignore the x coordinate.
+{% endnote %}
 
-```xml
-<mapping:rbf-thin-plate-splines direction="read" from="MyMesh2" to="MyMesh1" constraint="consistent"/>
-```
+* `rbf-pum-direct`, which breaks down the mapping problem in smaller clusters, solves these clusters locally and blends them afterwards together to recover a global solution. This mapping version only needs the linear-algebra library Eigen and the used linear solver is a dense solver in each cluster (actually the same as for `rbf-global-direct`, i.e., it is beneficial to configure strictily positive definite basis-functions). The mapping is specificially designed for large mapping problems and runs fully mpi-parallel. To configure the accuracy of the mapping, the number of `vertices-per-cluster` can be increased. The method can only handle sufficiently matching geometries and problems might occur if larger gaps exist between the coupling meshes. Further information, including some performance comparisons, can be found in the [talk of the preCICE workshop 2023](https://youtu.be/df-JMl7UxRg?si=18X3LFTIepmrtAMc).
```

Comment:
Should we mention that PUM is much faster than global RBF or similar?

Reply:
I added a hint at the end. People should anyway use the alias. Feel free to edit though.